lib/zclient.c

```diff
@@ -4252,3 +4257,28 @@ int zclient_neigh_ip_decode(struct stream *s, struct zapi_neigh_ip *api)
  stream_failure:
 	return -1;
 }
+
+int tun_send_zebra_gre_request(struct zclient *client,
+			       struct interface *ifp,
+			       uint32_t *gre_ikey,
+			       uint32_t *gre_okey,
+			       unsigned int *link_index,
+			       vrf_id_t *link_vrf_id,
+			       struct in_addr *addr)
```

Comment:
other than 'ifp', none of these arguments are used?

Reply:
oh yes.
this is a primary version that was used to be syncronous.
but as it is asyncronous, no need of those params.
thanks.